tao_compiler/mlir/xla/ral/context/common_context_impl_quantization.cc

```diff
@@ -726,6 +725,102 @@ void ral_qgemm_onednn_s8_s8_s8_per_channel(
   timer.Stop();
 }
 
+MemRefType<int8_t, 2> ral_pdll_qgemm_onednn_s8_s8_s8_per_channel(
+    ExecutionContext* ctx, opaque_t /*stream_handle*/,
+    MemRefType<int8_t, 2> input, MemRefType<int8_t, 2> weight,
+    MemRefType<float, 1> bias, MemRefType<float, 0> inputScales,
```

Comment:
so the bias input is not quantized as other inputs?

Reply:
yes, leave it to oneDNN to convert it s32 using the scale we passed to